src/include/duckdb/execution/window_segment_tree.hpp

```diff
@@ -61,6 +62,12 @@ class WindowSegmentTree {
 	//! The (sorted) input chunk collection on which the tree is built
 	ChunkCollection *input_ref;
 
+	//! Use the window API, if available
+	bool use_window_api;
```

Comment:
Perhaps store only the `WindowAggregationMode` and turn these into functions (`UseWindowAPI()`, `UseCombineAPI()`)?

Reply:
Sure. I did it this way because my temporary hack used the Booleans.